site/content/docs/cli/constructs-reference.md

```diff
@@ -94,6 +94,7 @@ used for form encoded payloads. The example below shows the following:
   - the JSON response body has a property called `name` by using the [JSON path](https://jsonpath.com/) expression `$.name`
   - the `strict-transport-security` response header's `max-age` property has a value greater than 100000.
 - It runs a **setup script** and **teardown script**, which are just TypeScript files referenced from the same directory.
+- It determines that the check should pass with `shouldFail: false`.
```

Comment:
@modern-sapien the `shouldFail` option influences the [request itself](https://www.checklyhq.com/docs/api-checks/request-settings/#method--url), not the check as a whole. Let's make that clear.

In general: instead of using `shouldFail: false` in the example (I'm assuming `false` here is the default), let's just list the parameter in the list below. I don't think that this not being in the example takes anything away.

Reply:
Done